src/utils/access.ts

```diff
@@ -53,3 +54,20 @@ export const checkRoleForAccess = (userRole: UserRole | null, operation: AccessO
     if (userRole?.[operation]) return allowed;
     return notAllowed(`${tr('Operation')} '${operation}' ${tr('is forbidden')}`);
 };
+
+export const scopes: AccessOperation[] = objKeys({
```

Comment:
This type won't show you an error if you forget one of the scopes.
I guess we need two variables to get type-safety: `scopesObj: Record<AccessOperation, true>` and then `scopes: AccessOperation[] = objKeys(...)`